sites/public/src/components/shared/MapControl.module.scss

```diff
@@ -0,0 +1,32 @@
+$control-border-radius: 100px;
+
+$control-background-color: white;
+
+.map-control {
+  z-index: 1;
+  display: inline-block;
+  flex: 1;
+  flex-direction: column;
+  position: absolute;
+  bottom: 10%;
+  left: 5%;
+}
+
+.control-style {
```

Comment:
can we add some padding here? imo `padding: var(--control-button-padding)` where the var  is `--bloom-s1` looks pretty good to me

Reply:
I'm not seeing an advantage of defining local variables used once as a variable referencing a global variable. I can't imagine a use case where we'd want to change the global to affect a single component; and if the variable is defined locally, once, declaring a variable is superfluous.

Reply:
The advantage of defining a local variable is being able to reuse it within this file, and also for consistency since that's done elsewhere in bloom. I think consistency/precedent is enough of an argument. For example --bloom-border-1 is used repeatedly throughout this file, it would be better if you had a local variable called control-border and then assigned it a value of var(--bloom-border-1) and used that instead. Also, can you define variables using -- instead of $ and then use them using var()?